### PR TITLE
Shruti Assignment 1 - 90mints

### PR DESCRIPTION
**Pull request by ShrutiDJun20**

### CODE REVIEW

File: src/shruti/FormData.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+			WebElement firstName = driver.findElement(By.id("first name"));
+			firstName.sendKeys("Shruti");
+			WebElement lastName = driver.findElement(By.id("last name"));
+			lastName.sendKeys("Dubey");
+			WebElement email = driver.findElement(By.id("E-mail"));
+			email.sendKeys("shrutidubey@gmail.com");
+			WebElement companyName = driver.findElement(By.id("Company Name"));
+			companyName.sendKeys("Infogain");
+
+			WebElement gender = driver.findElement(By.id("maleG"));
```
Comment (KrishnaKTechnocredits) [inline src/shruti/FormData.java:27]: You Swap the validation, for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’ radio button and vice versa.

File: src/shruti/FormData.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+			if (firstName.getAttribute("value").equals("Shruti")
+					&& lastName.getAttribute("value").equals("Dubey")) {
+				System.out.println("Value is entered correctly");
+			} else
+				System.out.println("Value is not entered correctly");
+			
+			// when Reset is clicked
+			driver.findElement(By.id("resetBtn")).click();
+			if (driver.findElement(By.id("first name")).getAttribute("value")
```
Comment (KrishnaKTechnocredits) [inline src/shruti/FormData.java:46]: Why here you have not used already created webElement "firstName".

File: src/shruti/FormData.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+		void dataEntryinForm() {
+			System.setProperty("webdriver.chrome.driver",
+					".\\Resources\\chromedriver.exe");
+			WebDriver driver = new ChromeDriver();
+			driver.get("file:///D:/Technocredit/Selenium%20recordings/@Selenium/SeleniumAssignment_1.html");
+			WebElement firstName = driver.findElement(By.id("first name"));
+			firstName.sendKeys("Shruti");
+			WebElement lastName = driver.findElement(By.id("last name"));
+			lastName.sendKeys("Dubey");
+			WebElement email = driver.findElement(By.id("E-mail"));
```
Comment (KrishnaKTechnocredits) [inline src/shruti/FormData.java:22]: You should only store any value in webElement when you want to perform more than 1 action on it..
But here you are only performing one action which sendKeys() so you should write like driver.findElement(By.id("E-mail")).sendKeys("shrutidubey@gmail.com");

File: src/shruti/FormData.java (hunk 1 above)
Comment (ShrutiDJun20) [reply src/shruti/FormData.java:27]: Changes done as per comment
image

File: src/shruti/FormData.java (hunk 2 above)
Comment (ShrutiDJun20) [reply src/shruti/FormData.java:46]: changes done as per comment..will keep in mind next time.. thanks

File: src/shruti/FormData.java (hunk 3 above)
Comment (ShrutiDJun20) [reply src/shruti/FormData.java]: changes done..Thanks

**Pull request merged**